ui/widgets/nuxeo-directory-suggestion.js

```diff
@@ -104,6 +105,12 @@ import { escapeHTML } from './nuxeo-selectivity.js';
          */
         dbl10n: { type: Boolean, value: false },
 
+        /**
+         * Checking this option means that the labels are localized with translations from JSON
+         * properties.
+         */
+        localize: { type: Boolean, value: true },
```

Comment:
Since the property is boolean and and the default is set to true, how are people supposed to set it to false?

Reply:
@tmartins , with this set to true by default, people won't be able to toggle it off using markup, which means that this won't solve the problem.